global_helpers/global_helpers_test.py

```diff
@@ -1229,6 +1229,55 @@ def test_deep_get(self):
         # If the searched key is not found, and no default kwarg is provided, return None
         self.assertEqual(p_b_h.deep_get(event, "key_does_not_exist"), None)
 
+    def test_deep_get_lists(self):
+        event = {
+            "key": {
+                "inner_key": [{"nested_key": "value"}],
```

Comment:
```suggestion
                "inner_key": [
                   {"nested_key": "value"},
                   {"nested_key": "value2"}
                  ],
```

Reply:
Updated this test case in [`5d5949a` (#804)](https://github.com/panther-labs/panther-analysis/pull/804/commits/5d5949a3e33724d3f7e3d93bea03ab87461028c2).